tests/unit/s2n_security_policies_test.c

```diff
@@ -190,6 +199,7 @@ int main(int argc, char **argv)
             "20190120",
             "20190121",
             "20190122",
+            "20201010",
             "test_all_fips",
             "test_all_ecdsa",
```

Comment:
Should this case (`test_all_ecdsa`) be modified too?

For that matter, why are we adding support for P-512 on key agreement but not on signatures? That encourages an algorithmic mismatch (often a bad idea).

Reply:
add P-521 to signature as well and update security polices for new curve.